src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -177,18 +177,10 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
     this.httpRequestService.post('/course/enrollSave', paramMap, this.enrollData)
         .subscribe((resp: EnrollResultPanelList) => {
           this.showEnrollResults = true;
-          this.statusMessage.pop(); // removes any existing status message
-          this.statusMessage.push({
-            message: 'Enrollment Successful. Summary given below',
-            color: 'success',
-          });
+          this.statusMessageService.showSuccessMessage('Enrollment successful. Summary given below.');
```

Comment:
Same here.

Reply:
Since this message is short, I think the snackbar is suitable. Should I just use the normal status message only for the long chunk of error message related to HOT?

Reply:
Yup, that's fine 👍